.cirrus.yml

```diff
@@ -53,6 +39,26 @@ low_scale_task:
         apt update && apt install -y ${DEPENDENCIES};
        fi'
 
+runtime_builder_task:
+  << : *COMMON_TASK_TEMPLATE
+
+  compute_engine_instance:
+    matrix:
+      - image_project: fedora-cloud
+        image: family/fedora-cloud-38
+      - image_project: ubuntu-os-cloud
+        image: family/ubuntu-2304-amd64
+    platform: linux
+    memory: 8G
+    disk: 40
+
+  runtime_cache:
+    folder: runtime-cache
+    fingerprint_script:
+      - echo ${CIRRUS_BUILD_ID}
```

Comment:
This is a little concerning.   We could consider going back to a simple loop, but we could also consider having a separate global and local to the job caches to save CPU cycles.

Reply:
I'll remove the commit that increases CI coverage from this PR.  We need the address set fix sooner rather than later.  I'll open an issue to track adding global and local to the job caches to save CPU cycles.